x-pack/plugins/triggers_actions_ui/public/application/components/builtin_action_types/email/email_connector.tsx

```diff
@@ -160,60 +167,95 @@ export const EmailActionConnectorFields: React.FunctionComponent<ActionConnector
           </EuiFlexGroup>
         </EuiFlexItem>
       </EuiFlexGroup>
-      <EuiFlexGroup justifyContent="spaceBetween">
-        <EuiFlexItem>
-          <EuiFormRow
-            id="emailUser"
-            fullWidth
-            error={errors.user}
-            isInvalid={errors.user.length > 0}
-            label={i18n.translate(
-              'xpack.triggersActionsUI.sections.builtinActionTypes.emailAction.userTextFieldLabel',
-              {
-                defaultMessage: 'Username',
-              }
-            )}
-          >
-            <EuiFieldText
-              fullWidth
-              isInvalid={errors.user.length > 0}
-              name="user"
-              readOnly={readOnly}
-              value={user || ''}
-              data-test-subj="emailUserInput"
-              onChange={(e) => {
-                editActionSecrets('user', nullableString(e.target.value));
-              }}
-            />
-          </EuiFormRow>
-        </EuiFlexItem>
+      <EuiFlexGroup>
         <EuiFlexItem>
-          <EuiFormRow
-            id="emailPassword"
-            fullWidth
-            error={errors.password}
-            isInvalid={errors.password.length > 0}
-            label={i18n.translate(
-              'xpack.triggersActionsUI.sections.builtinActionTypes.emailAction.passwordFieldLabel',
-              {
-                defaultMessage: 'Password',
-              }
-            )}
-          >
-            <EuiFieldPassword
-              fullWidth
-              readOnly={readOnly}
-              isInvalid={errors.password.length > 0}
-              name="password"
-              value={password || ''}
-              data-test-subj="emailPasswordInput"
+          <EuiFormRow hasEmptyLabelSpace>
+            <EuiSwitch
+              label={i18n.translate(
+                'xpack.triggersActionsUI.sections.builtinActionTypes.emailAction.noCredentialsSwitchLabel',
+                {
+                  defaultMessage: 'Do not send credentials',
```

Comment:
Left a discussion / comment here about the label: https://github.com/elastic/kibana/issues/74870#issuecomment-697819895.